admin_guide/scheduler.adoc

```diff
@@ -214,18 +214,26 @@ they have to be explicitly specified in the scheduler configuration file.
 === Default Scheduler Policy
 The default scheduler policy includes the following predicates:
 
-. PodFitsPorts
-. PodFitsResources
 . NoDiskConflict
-. MatchNodeSelector
-. HostName
+. NoVolumeZoneConflict
+. MaxEBSVolumeCount
+. MaxGCEPDVolumeCount
+. GeneralPredicates
```

Comment:
I guess GeneralPredicates could be expanded to:

- noncritical Predicates(that are needed for non critical pods only. Eg: PodFitsResources).
-Essential Predicates(that are needed for every pod including critical ones: Eg: PodFitsHost, PodFitsHostPorts, PodSelectorMatches).